tests-extended/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+project(ExtendedTests)
```

Comment:
While we're at it, let's restructure the repo so that all tests are in `tests/` directory, notably the `db/` directory.
Again, check similar transition we already did in our sister project SCC for details.

Reply:
That is an excellent idea. Shall I move bandit to tests as well? Or keep him where he is?

Reply:
I've considered it - IMHO, it's better to not "encapsulate" the external dependencies that way - better to keep them visible on top level.